src/StableHashTraits.jl

```diff
@@ -296,6 +304,9 @@ hash_method(::Missing) = UseQualifiedName()
 hash_method(::VersionNumber) = UseProperties()
 hash_method(::UUID) = UseProperties()
 hash_method(::Dates.AbstractTime) = UseProperties()
+# TODO: improve the API so context knows about the algorithm
+# c.f. https://github.com/beacon-biosignals/StableHashTraits.jl/issues/17
+transform(x::Set, context) = sort!(map(xᵢ -> stable_hash(xᵢ; context), collect(x)))
```

Comment:
We can avoid the extra allocation with:
```suggestion
transform(x::Set, context) = sort!([stable_hash(xᵢ; context) for xᵢ in x])
```